openeo/udf/xarraydatacube.py

```diff
@@ -156,18 +156,18 @@ def save_to_file(self, path: Union[str, Path], fmt=None, **kwargs):
             raise ValueError(fmt)
 
     def plot(
-            self,
-            title: str = None,
-            limits=None,
-            show_bandnames: bool = True,
-            show_dates: bool = True,
-            show_axeslabels: bool = False,
-            fontsize: float = 10.,
-            oversample: float = 1,
-            cmap: Union[str, 'matplotlib.colors.ColorMap'] = 'RdYlBu_r',
-            cbartext: str = None,
-            to_file: str = None,
-            to_show: bool = True
+        self,
+        title: str = None,
+        limits=None,
+        show_bandnames: bool = True,
+        show_dates: bool = True,
+        show_axeslabels: bool = False,
+        fontsize: float = 10.0,
+        oversample: float = 1,
+        cmap: Union[str, matplotlib.colors.Colormap] = "RdYlBu_r",
```

Comment:
Note: this line broke building the docs.
Apparently it is a misspelling of Colormap and sphinx_autodoc_typehints raised an error.

Otherwise I would avoid changing code when working a issue about updating docs.

Commit: https://github.com/Open-EO/openeo-python-client/pull/462/commits/85252b8a0632353082683d14adecf9b2196f0dbe

Reply:
I already fixed it on master in 6bb1eda811386c14f0fba7eef35ad7b21270c744

note that we can not remove the string version of the annotation because matplotlib is not a dependency of the openeo library (optional at most)

Reply:
this branch will now probably conflict with master. 
But if you are familiar with that: you could just rebase interactively and drop your fix 85252b8
and force-push your feature branch

Reply:
Indeed, fixing this on the master branch is better.
I rebased and dropped commit https://github.com/Open-EO/openeo-python-client/commit/85252b8a0632353082683d14adecf9b2196f0dbe